modules/AlphaWalletWeb3/AlphaWalletWeb3/Promises/Promise+Batching.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import PromiseKit
 
 public class JSONRPCrequestDispatcher {
-    public var MAX_WAIT_TIME: TimeInterval = 0.1
+    public var MAX_WAIT_TIME: TimeInterval = 0.5
```

Comment:
Have you try measuring/logging to get a feel how many requests this cuts down to?

Reply:
Also: https://github.com/AlphaWallet/alpha-wallet-ios/issues/6540#issuecomment-1498346374